Dockerfile

```diff
@@ -1,10 +1,33 @@
-FROM debian:squeeze
+FROM ubuntu:xenial
+
+# RUN COMMAND
 
-# Add Compiled Gogeta 
 RUN mkdir -p /var/www/go/bin; \
-    mkdir -p /var/www/go/big/client/logs
+    mkdir libgit2
+
+RUN apt-get -y update; \
+    apt-get -y upgrade; \
+    apt-get -y install cmake; \
+    apt-get -y install git; \
```

Comment:
work around for lack of git clone --depth=2 ; trade off is smaller repos to copy around.